test/core/end2end/invalid_call_argument_test.cc

```diff
@@ -607,6 +607,31 @@ static void test_invalid_initial_metadata_reserved_key() {
   cleanup_test();
 }
 
+static void test_multiple_recv_initial_metadata_in_a_single_batch() {
```

Comment:
Looks like this actually tests duplicates of all ops, not just recv_initial_metadata.

Reply:
fixed